backend/models/comment.js

```diff
@@ -20,4 +20,11 @@ commentSchema.method("update", function(updates, callback) {
   Object.assign(this, updates, {updatedAt: new Date()});
   this.save(callback);
 });
-module.exports = mongoose.model('Comment', commentSchema)
+commentSchema.statics.getComment = function(request, response, next, id) {
+  return new Promise(async (resolve, reject) => {
+    const doc = await Comment.findById(id)
+    resolve(doc)
+  })
+}
+var Comment = mongoose.model('Comment', commentSchema)
+module.exports = Comment
```

Comment:
this is some interesting code. Although, I haven't delved deeper into mongo. I would appreciate it if you could explain to me this code `[Schema].statics.[method]`?

Reply:
It's just a way to add a method to a specific schema. You could implement the same functionality inside routes/post.js, but keeping it attached to the schema makes it easier to read and maintain.